solutions.sql

```diff
@@ -0,0 +1,84 @@
+Step 1
+CREATE DATABASE books;
+Step2
+\c books;
+Step3
+\i /Users/jisuyeo/Documents/ga/homworks/booktown/seed.sql
+\i /Users/jisuyeo/Documents/ga/homworks/booktown/schema.sql
+id |           name
+----+--------------------------
+  1 | King    Stephen
+  2 | Herbert Frank
+  3 | Clarke  Arthur C.
+  4 | Bourgeois       Paulette
+  5 | Brown   Margaret Wise
+  6 | Alcott  Louisa May
+  7 | Bianco  Margery Williams
+  8 | Hogarth Burne
+  9 | Lutz    Mark
+
+Q1
+SELECT books.title FROM 
+books JOIN authors ON books.authorId = authors.id
+WHERE authors.name ='Lutz	Mark';
+Q2
+SELECT books.title FROM
+books JOIN subjects ON books.subjectId = subjects.id
+WHERE subjects.name = 'Childrens Books';
+Q3
+-- SELECT subjects.name FROM 
+-- books JOIN subjects ON books.subjectId = subjects.id
+-- WHERE books.subjectId = _;
+Q4
+-- 
```

Comment:
Question 4 was too get all authors that have written books on a given subject

 So we'd SELECT authors.name 
FROM authors and to join 

"JOIN books ON authors.id = books.authorId
 JOIN subjects ON subjects.id = books.subjectId"